ratpack-core/src/main/java/ratpack/http/client/internal/RequestActionSupport.java

```diff
@@ -166,10 +167,17 @@ private void finalizeHeaders() {
     }
     int contentLength = requestConfig.body.readableBytes();
     if (contentLength > 0) {
-      requestConfig.headers.set(HttpHeaderConstants.CONTENT_LENGTH, Integer.toString(contentLength));
+      // See https://github.com/netty/netty/pull/9865/files#diff-bf7a179a319b29e0262d5badaf3770e0R634-R650
```

Comment:
After the upgrade these tests were failing, the following changes made them pass again:
<img src="https://user-images.githubusercontent.com/1841944/74530446-5a465180-4f2a-11ea-9546-962ad6d773ba.png">